storage/service_account/hmac/hmac_test.go

```diff
@@ -177,7 +178,8 @@ func createTestKey(projectID string, t *testing.T) (*storage.HMACKey, error) {
 		// Nil key check should not happen but is added to handle flaky
 		// "nil pointer dereference" error.
 		if key == nil {
-			r.Errorf("Returned nil key.")
+			r.Errorf("CreateHMACKey returned nil key.")
+			err = errors.New("CreateHMACKey returned nil key")
```

Comment:
is it worth doing `r.Fatal(err)` instead of copying the string literal?
Also not sure if this should be `r.Fatalf()` instead of `r.Errorf()`

Reply:
Unfortunately `testutil.R` does not have a `Fatalf`. Maybe we should add it!

Reply:
Ah gotcha, wasn't sure, ignore this comment ty